src/test/scala/mesosphere/marathon/util/StreamHelpers.scala

```diff
@@ -0,0 +1,12 @@
+package mesosphere.marathon
+package util
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+
+object StreamHelpers {
+  /**
+    * Source which conforms to any type; never completes and never emits anything.
+    */
+  val sourceNever: Source[Nothing, NotUsed] = Source.maybe[Nothing].mapMaterializedValue { _ => NotUsed }
```

Comment:
We should use this in https://github.com/mesosphere/marathon/blob/master/src/test/scala/mesosphere/marathon/core/task/termination/impl/KillStreamWatcherTest.scala#L24 as well.